app/facades/food_facade.rb

```diff
@@ -0,0 +1,13 @@
+class FoodFacade
```

Comment:
Typically, facade and services use class methods so that you don't need to instantiate them!

They are sort of utility classes (they do stuff for us) rather than classes that need to hold any state (like a PORO holding state in attributes). That makes it more reasonable for these methods to be class methods rather than instance methods